.eslintrc.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
     ignorePatterns: [
         'dist',
         '*schema.ts',
+        ...['condo', 'callcenter'].map(app => `apps/${app}/gql/*.ts`),
```

Comment:
Why only 2 of apps, not all of them?

Reply:
You are right, it is better to add it everywhere at once